nemo/collections/llm/recipes/llama3_70b.py

```diff
@@ -44,6 +45,8 @@ def trainer(
         virtual_pipeline_model_parallel_size=virtual_pipeline_parallelism,
         context_parallel_size=context_parallelism,
         sequence_parallel=sequence_parallelism,
+        tp_comm_overlap=False,
```

Comment:
Can we make bundle both tp_comm_overlap and  
tp_comm_overlap_cfg into one dataclass ?

```
class TPCommOverlap: 
     enabled : Default(False)
     cfg_settings : Optional[]

```

Why : Helped keep all the relevant info together (Encapsulation)